jormungandr-integration-tests/src/mock/client.rs

```diff
@@ -0,0 +1,233 @@
+extern crate base64;
+extern crate chain_impl_mockchain;
+extern crate futures;
+extern crate futures_cpupool;
+extern crate grpc;
+extern crate hex;
+extern crate protobuf;
+
+use crate::mock::{
+    grpc::*,
+    proto::{node::*, node_grpc::*},
+    read_into,
+};
+use chain_core::property::FromStr;
+use chain_core::property::Serialize;
+use chain_impl_mockchain as chain;
+use chain_impl_mockchain::{
+    block::{Block as ChainBlock, Header as ChainHeader},
+    fragment::Fragment as ChainFragment,
+    key::Hash,
+};
+use protobuf::RepeatedField;
+
+pub struct JormungandrClient {
+    client: NodeClient,
+    host: String,
+    port: u16,
+}
+
+impl Clone for JormungandrClient {
+    fn clone(&self) -> Self {
+        JormungandrClient::new(&self.host, self.port)
+    }
+}
+
+impl JormungandrClient {
+    pub fn new(host: &str, port: u16) -> Self {
+        let client_conf = Default::default();
+        let client = NodeClient::new_plain(host, port, client_conf).unwrap();
+        Self {
+            client: client,
+            host: host.to_owned(),
+            port: port,
+        }
+    }
+
+    pub fn get_blocks(&self, blocks_id: &Vec<Hash>) -> Vec<ChainBlock> {
+        self.get_blocks_stream(blocks_id)
+            .into_future()
+            .wait_drop_metadata()
```

Comment:
Now I remember again why I rejected that gRPC stack, it's so fugly.
There is also another one that binds to the C library, check if that might be better.

On the other hand, it could be useful for dissimilar interop testing.